variables.tf

```diff
@@ -35,3 +35,30 @@ variable "region" {
     error_message = "Invalid input, options: \"us-south\", \"ca-tor\", \"eu-es\", \"eu-de\"."
   }
 }
+
+variable "cos_instance_guid" {
```

Comment:
There is no need to have this variable. We can parse the GUID from the CRN using a local variable:

`cos_instance_guid = element(split(":", var.cos_instance_crn), length(split(":", var.cos_instance_crn)) - 3) : null`